### PR TITLE
ENG-555: home hero particles

### PR DESCRIPTION
**Pull request by stormcloud266**

### CODE REVIEW

File: lib/animations/hero-stars.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        speed: 3,
+      },
+      grab: { distance: 400, line_linked: { opacity: 1 } },
+      push: { particles_nb: 4 },
+      remove: { particles_nb: 2 },
+      repulse: { distance: 400, duration: 0.4 },
+    },
+  },
+  particles: {
+    color: { value: "#3FCCBB" },
```
Comment (illepic) [inline lib/animations/hero-stars.ts:32]: Use tailwind vars here.

File: lib/animations/hero-stars.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    line_linked: {
+      color: "#ffffff",
```
Comment (illepic) [inline lib/animations/hero-stars.ts:34]: Tailwind var all the colors.

**Review by illepic — CHANGES_REQUESTED** [review]

File: lib/animations/hero-stars.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
+import type { IOptions, RecursivePartial } from "tsparticles-engine";
+import resolveConfig from "tailwindcss/resolveConfig";
+import tailwindConfig from "../../tailwind.config.js";
+
+const fullConfig = resolveConfig(tailwindConfig) as any;
+const { white, black, turquoise } = fullConfig.theme?.colors;
```
Comment (stormcloud266) [inline lib/animations/hero-stars.ts]: Removing "as any" gives me this error for every color, and every other property I tried.

```
Property 'black' does not exist on type 'ResolvableTo<RecursiveKeyValuePair<string, string>> | undefined'.
```

Ended up going [this route](https://github.com/tailwindlabs/tailwindcss/issues/6422#issuecomment-1222368084), I couldn't find any better option

**Comment by stormcloud266** [comment]
The [tailwind docs](https://tailwindcss.com/docs/configuration#referencing-in-java-script) mentioned using babel-plugin-preval when using the config in js, is that something I should add here?

**Comment by illepic** [comment]
resolveConfig here: https://medium.com/pragmatic-programmers/access-tailwind-from-javascript-8e97f3c92ca6

**Pull request merged**